### ISSUE TITLE
Delete Elements:  removeElements(elementsToRemove, els)

### ISSUE THREAD

**Issue by monikabadertscher** [wbkd/react-flow#1764]
Hi 
In a certain situation I want to remove all edges to and from a certain node programmatically and I do it with  removeElements(elementsToRemove, els). Interestingly enough all edges are removed, but after updating the node 1 edge to this node will still (or again) be there. Is this behaviour known or is there something special to consider when applying this function?
Thank you,
Moka Badertscher

**Comment by moklick** [comment]
Hey @monikabadertscher 

sorry but I don't get the issue and I can't reproduce it. Can you send a repo or codesandbox that illustrates the issue?

**Comment by monikabadertscher** [comment]
Hi Moritz
Thank you for your answer. This afternoon I found a way to solve it and what the problem was.
When I just moved or added handles, the connections where removed automatically by the react-flow-renderer framework, but then added again (interestingly enough only one of them). So I tried to remove the connectors programmatically, but did something wrong.
I tried it again today and I saw that I used the remove function in a wrong way. Now it works fine.
Thank you much and I appreciate your service!
Moka
